test/controllers/AmbiguousJourneyControllerSpec.scala

```diff
@@ -62,6 +60,7 @@ class AmbiguousJourneyControllerSpec extends BaseSpec {
       def nino: Nino
       def personDetailsResponse: PersonDetailsResponse
       def getSelfAssessmentServiceResponse: SelfAssessmentUserType
+      def saSkipPage : Boolean
```

Comment:
`false` is used a lot as a default value, you could add that here and remove it from other tests